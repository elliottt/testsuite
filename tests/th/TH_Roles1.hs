{-# LANGUAGE TemplateHaskell #-}

module TH_Roles1 where

import Language.Haskell.TH

$( return [DataD [] (mkName "T") [RoledTV (mkName "a") Representational] [] []] )

