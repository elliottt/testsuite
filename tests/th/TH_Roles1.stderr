
TH_Roles1.hs:7:4:
    Illegal role annotation
    Perhaps you intended to use -XRoleAnnotations
    In the data type declaration for ‛T’
